core/src/main/java/ai/timefold/solver/core/impl/localsearch/decider/acceptor/restart/RestartStrategy.java

```diff
@@ -0,0 +1,11 @@
+package ai.timefold.solver.core.impl.localsearch.decider.acceptor.restart;
+
+import ai.timefold.solver.core.impl.localsearch.event.LocalSearchPhaseLifecycleListener;
+import ai.timefold.solver.core.impl.localsearch.scope.LocalSearchMoveScope;
+
+public interface RestartStrategy<Solution_> extends LocalSearchPhaseLifecycleListener<Solution_> {
```

Comment:
Needs to be Javadoc'd.
Also arguably better naming - what does it mean to be "triggered"? What does "reset" do?